lib/v2/screens/passcode/components/create_passcode.dart

```diff
@@ -0,0 +1,56 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:passcode_screen/circle.dart';
+import 'package:passcode_screen/passcode_screen.dart';
+import 'package:provider/provider.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/features/biometrics/auth_commands.dart';
+import 'package:seeds/i18n/passcode.i18n.dart';
+import 'package:seeds/features/biometrics/auth_bloc.dart';
+import 'package:seeds/v2/screens/passcode/interactor/viewmodels/bloc.dart';
+
+class CreatePasscode extends StatefulWidget {
+  const CreatePasscode({Key key}) : super(key: key);
+
+  @override
+  _CreatePasscodeState createState() => _CreatePasscodeState();
+}
+
+class _CreatePasscodeState extends State<CreatePasscode> {
+  final StreamController<bool> _verificationNotifier = StreamController<bool>.broadcast();
+
+  @override
+  Widget build(BuildContext context) {
+    var bloc = Provider.of<AuthBloc>(context);
+    return PasscodeScreen(
```

Comment:
please, check that the UI settings match what we use in onboarding, 
backgroundColor
bottomWidget
etc

Reply:
Which ones do not match? It is only the white and the primary colors.

The button will stay as is according to Jason.

Reply:
My comment doesnt say they dont match. 

My comments asks to check that they match :)